packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -22,9 +22,9 @@ type Props = {|
      * A URL.
      *
      * If specified, we will assume the rendered component should react like an
-     * `<a>` tag and respond to enter/return key press. Otherwise, we will
-     * assume the rendered component should react like a `<button>` tag and
-     * respond to spacebar key press.
+     * `<a>` tag and respond to enter/return key press but not space key press.
+     * If not specified, the component will react to both enter and space key
+     * presses.
```

Comment:
How does this jibe with want the same behaviour for things that look like buttons but actually use <a> or <button> under the hood?

Reply:
It might be worth calling out that `triggerOnEnter` and `triggerOnSpace` override this behaviour.

Reply:
Ooh I forgot to update this comment, thanks! (This was the previous way of doing this.)